rotkehlchen/accounting/constants.py

```diff
@@ -1,206 +1,264 @@
+        HistoryEventSubType.REWARD: EventCategory.STAKING_REWARD,
         HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
-        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
-        HistoryEventSubType.MEV_REWARD: EventCategory.MEV_REWARD,
         HistoryEventSubType.BLOCK_PRODUCTION: EventCategory.CREATE_BLOCK,
-        HistoryEventSubType.REWARD: EventCategory.STAKING_REWARD,
+        HistoryEventSubType.MEV_REWARD: EventCategory.MEV_REWARD,
+    },
+    HistoryEventType.TRANSFER: {
+        HistoryEventSubType.DONATE: EventCategory.DONATE,
+        HistoryEventSubType.NONE: EventCategory.TRANSFER,
+    },
+    HistoryEventType.UNKNOWN: {
+        HistoryEventSubType.NONE: EventCategory.INFORMATIONAL,
+    },
+    HistoryEventType.ADJUSTMENT: {
+        HistoryEventSubType.SPEND: EventCategory.SEND,
+        HistoryEventSubType.RECEIVE: EventCategory.RECEIVE,
     },
 }
 
 EVENT_CATEGORY_DETAILS = {
     EventCategory.GAS: EventCategoryDetails(
         label='gas fee',
         icon='mdi-fire',
+        direction=EventDirection.OUT,
     ),
     EventCategory.SEND: EventCategoryDetails(
         label='send',
         icon='mdi-arrow-up',
+        direction=EventDirection.OUT,
     ),
     EventCategory.RECEIVE: EventCategoryDetails(
         label='receive',
         icon='mdi-arrow-down',
         color='green',
+        direction=EventDirection.IN,
     ),
     EventCategory.SWAP_OUT: EventCategoryDetails(
         label='swap out',
         icon='mdi-arrow-u-right-bottom',
+        direction=EventDirection.OUT,
     ),
     EventCategory.SWAP_IN: EventCategoryDetails(
         label='swap in',
         icon='mdi-arrow-u-left-top',
         color='green',
+        direction=EventDirection.IN,
     ),
     EventCategory.APPROVAL: EventCategoryDetails(
         label='approval',
         icon='mdi-lock-open-outline',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.DEPOSIT: EventCategoryDetails(
         label='deposit',
         icon='mdi-arrow-expand-up',
         color='green',
+        direction=EventDirection.OUT,
     ),
     EventCategory.WITHDRAW: EventCategoryDetails(
         label='withdraw',
         icon='mdi-arrow-expand-down',
+        direction=EventDirection.IN,
     ),
     EventCategory.AIRDROP: EventCategoryDetails(
         label='airdrop',
         icon='mdi-airballoon-outline',
+        direction=EventDirection.IN,
     ),
     EventCategory.BORROW: EventCategoryDetails(
         label='borrow',
         icon='mdi-hand-coin-outline',
+        direction=EventDirection.IN,
     ),
     EventCategory.REPAY: EventCategoryDetails(
         label='repay',
         icon='mdi-history',
+        direction=EventDirection.OUT,
     ),
     EventCategory.DEPLOY: EventCategoryDetails(
         label='deploy',
         icon='mdi-swap-horizontal',
+        direction=EventDirection.NEUTRAL,
     ),
     EventCategory.BRIDGE_IN: EventCategoryDetails(
```

Comment:
Can we rename this. BRIDGE_DEPOSIT? I think BRIDGE_IN is quite confusing. Same with BRIDGE_OUT below

Reply:
Resolved with suggested change - with categories BRIDGE_DEPOSIT and BRIDGE_WITHDRAWAL.